Command/ClearOrphansCommand.php

```diff
@@ -2,14 +2,24 @@
 
 namespace Oneup\UploaderBundle\Command;
 
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Oneup\UploaderBundle\Uploader\Orphanage\OrphanageManager;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
-class ClearOrphansCommand extends ContainerAwareCommand
+class ClearOrphansCommand extends Command
 {
     protected static $defaultName = 'oneup:uploader:clear-orphans'; // Make command lazy load
 
+    /** @var OrphanageManager */
+    protected $manager;
+
+    public function __construct(OrphanageManager $manager, ?string $name = null)
```

Comment:
New argument was introduced. It should be passed to command in `Resources/config/uploader.xml`.